data/players.yaml

```diff
@@ -5494,6 +5494,8 @@
       - "1829213"
       - "3038783"
       - "9736464"
+  twitch:
+    - https://www.twitch.tv/f47cao
```

Comment:
Is Falcao actually a streamer?

Reply:
I'm not sure if he intends to be a regular streamer, but today I don't believe he can be called a streamer. I added because I saw him at least 2 times in the last few days playing.

Reply:
Are you saying you saw him streaming or just playing?

Let's wait to add twitch IDs until at least one stream has happened.

Reply:
Hah well never mind, he started streaming at the same time I posed this comment so I got my answer :)